arroyo/processing/processor.py

```diff
@@ -435,6 +435,11 @@ def _run_once(self) -> None:
                         self.__consumer.pause([*self.__consumer.tell().keys()])
                         self.__is_paused = True
 
+                    elif self.__is_paused:
+                        # A paused consumer should still poll periodically to avoid it's partitions
+                        # getting revoked by the broker after reaching the max.poll.interval.ms
+                        # Polling a paused consumer should never yield a message.
+                        assert self.__consumer.poll(0.1) is None
```

Comment:
do we not want to sleep in this elif block?

Reply:
it's already got a sleep for `0.1` in there?